Cargo.toml

```diff
@@ -34,6 +34,7 @@ serde = { version = "1.0", features = ["derive"], optional = true }
 # https://github.com/rust-lang/cargo/issues/1596
 serde_json = { version = "1.0", optional = true }
 sqlparser_derive = { version = "0.1.1", path = "derive", optional = true }
+duplicate = "1.0.0"
```

Comment:
in general we try and keep the dependencies of this crate as minimal as possible -- can you please remove this dependency?

Reply:
got it!
please refer to it: [598d9e9](https://github.com/sqlparser-rs/sqlparser-rs/pull/870/commits/598d9e97ada700d1cf57b2ea17b383365dd2b542)